frontend/src/lib/components/ActivationSidebar/ActivationSidebar.tsx

```diff
@@ -0,0 +1,126 @@
+import { LemonButton, LemonButtonProps, LemonButtonWithSideAction } from '@posthog/lemon-ui'
+import clsx from 'clsx'
+import { useActions, useValues } from 'kea'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { activationLogic, ActivationTaskType } from './activationLogic'
+import './ActivationSidebar.scss'
+import { Progress } from 'antd'
+import { IconCheckmark, IconClose } from '../icons'
+import { SessionRecording as SessionRecordingConfig } from 'scenes/project/Settings/SessionRecording'
+import { ProfessorHog } from '../hedgehogs'
+
+const Task = ({ id, name, description, completed, canSkip, skipped, url }: ActivationTaskType): JSX.Element => {
+    const displaySideAction = !completed && !skipped && canSkip
+    const { runTask, skipTask } = useActions(activationLogic)
+
+    const content = (
+        <div className="my-4 mx-2">
+            <p className="m-0">{name}</p>
+            {!completed && !skipped && <p className="font-normal text-xs mt-2 mb-0">{description}</p>}
```

Comment:
For whatever reason all `<p>` tags have a 5px left margin. We should remove it here since it looks oddly indented.

<img width="389" alt="image" src="https://user-images.githubusercontent.com/18598166/204102715-7e406bdb-5423-4971-b492-a863eb7bd0f5.png">